global.bat

```diff
@@ -92,6 +92,7 @@ echo -----------------------------
 echo  Uploading YouTube HD stream 
 echo -----------------------------
 echo.
+start https://encoders.tasvideos.org/status.html
```

Comment:
This actually should happen in the next line. Append `^& start https://encoders.tasvideos.org/status.html ^& exit` instead of just exist.